.github/cilium-cli-test-job-chart/values.yaml

```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 
 tag: latest
-cilium_version: v0.10.10
+cilium_version: ""
```

Comment:
I can't find why this is updated in commit message, can you clarify ?

Reply:
It's set from the github actions explicitly, no need to specify the default value here as well.